File: apps/docs/content/guides/storage/examples/quickstart/sample.js
// Use the JS library to create a bucket.

const { data, error } = await supabase.storage.createBucket('avatars')

// 2. upload a file
const avatarFile = event.target.files[0]
const { data, error } = await supabase.storage
    .from('avatars')
    .upload('public/avatar1.png', avatarFile)


// 3. download a file.
const { data, error } = await supabase.storage.from('avatars').download('public/avatar1.png')

File: apps/docs/content/guides/storage/examples/quickstart/sample.dart
void main() async {
  final supabase = SupabaseClient('supabaseUrl', 'supabaseKey');

// 1. create a bucket
  final storageResponse = await supabase
      .storage
      .createBucket('avatars');

// 2. upload a file
final file = File('example.txt');
  file.writeAsStringSync('File content');
  final storageResponse = await supabase
      .storage
      .from('public')
      .upload('example.txt', file);

// 3. download a file
final storageResponse = await supabase
   .storage
   .from('public')
   .download('example.txt');

}


